test/functional/autocmd/termclose_spec.lua

```diff
@@ -25,4 +26,19 @@ describe('TermClose event', function()
       TermClose works!    |
     ]])
   end)
+
+  it('reports the correct <abuf>', function()
+    execute('set hidden')
+    execute('autocmd TermClose * let abuf = expand("<abuf>")')
```

Comment:
would you mind adding `g:` to globals? IMO it makes the tests much less prone to mistakes or confusion, given all the various contexts that are flying around with nvim sessions being created and destoryed, scripts being sourced, etc....